style.css

```diff
@@ -31,3 +34,29 @@ body {
   color: #423e3e;
   margin-top: 80px;
 }
+.score {
+  background-color: #edeff3;
+  width: max-content;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 10px;
+  border-radius: 5px;
+  margin-top: 30px;
+}
+
+.score__item {
+  display: inline-block;
+}
+.score__item:nth-child(3) {
```

Comment:
I suggest not to use `:nth-child(3)` for styling. This is too specific and highly dependent on HTML layout. Per BEM, it's recommended to use classes only.

As I understand, you want to add margin between all `score__item` elements. The same can be achieved by using an [adjacent sibling selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator)
```
.score__item + .score__item {
    margin-left: 10px;
}
```